### ISSUE TITLE
Spurious "When asked for SOA records with different cases, all servers do not reply consistently" 

### ISSUE THREAD

**Issue by bortzmeyer** [dotse/zonemaster#422]
In https://zonemaster.net/test/40350 I get "When asked for SOA records on "www.int-evry.fr" with different cases, all servers do not reply consistently."

And before that:

 When asked for SOA records on "WWw.iNt-Evry.Fr" and "Www.inT-EVRY.fr", nameserver shiva.jussieu.fr/134.157.0.129 returns different answers. 

To me, the answers *are* identical (the SOA differ in case but, for the DNS, they have the same canonical content):

```

% dig @shiva.jussieu.fr SOA Www.inT-EVRY.fr

; <<>> DiG 9.9.5-12-Debian <<>> @shiva.jussieu.fr SOA Www.inT-EVRY.fr
; (1 server found)
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 35035
;; flags: qr aa rd; QUERY: 1, ANSWER: 1, AUTHORITY: 1, ADDITIONAL: 1
;; WARNING: recursion requested but not available

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags: do; udp: 4096
;; QUESTION SECTION:
;Www.inT-EVRY.fr.       IN SOA

;; ANSWER SECTION:
Www.inT-EVRY.fr.        86400 IN CNAME www.it-sudparis.eu.

;; AUTHORITY SECTION:
it-sudparis.eu.         172800 IN SOA diamant.inT-EVRY.fr. admin.inT-EVRY.fr. (
                                2015101501 ; serial
                                21600      ; refresh (6 hours)
                                3600       ; retry (1 hour)
                                1209600    ; expire (2 weeks)
                                172800     ; minimum (2 days)
                                )

;; Query time: 2 msec
;; SERVER: 134.157.0.129#53(134.157.0.129)
;; WHEN: Mon Oct 19 13:28:07 CEST 2015
;; MSG SIZE  rcvd: 126

% dig @shiva.jussieu.fr SOA WWw.iNt-Evry.Fr

; <<>> DiG 9.9.5-12-Debian <<>> @shiva.jussieu.fr SOA WWw.iNt-Evry.Fr
; (1 server found)
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 51950
;; flags: qr aa rd; QUERY: 1, ANSWER: 1, AUTHORITY: 1, ADDITIONAL: 1
;; WARNING: recursion requested but not available

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags: do; udp: 4096
;; QUESTION SECTION:
;WWw.iNt-Evry.Fr.       IN SOA

;; ANSWER SECTION:
WWw.iNt-Evry.Fr.        86400 IN CNAME www.it-sudparis.eu.

;; AUTHORITY SECTION:
it-sudparis.eu.         172800 IN SOA diamant.iNt-Evry.Fr. admin.iNt-Evry.Fr. (
                                2015101501 ; serial
                                21600      ; refresh (6 hours)
                                3600       ; retry (1 hour)
                                1209600    ; expire (2 weeks)
                                172800     ; minimum (2 days)
                                )

;; Query time: 2 msec
;; SERVER: 134.157.0.129#53(134.157.0.129)
;; WHEN: Mon Oct 19 13:27:57 CEST 2015
;; MSG SIZE  rcvd: 126

**Comment by bortzmeyer** [comment]
Just tested the engine+CLI a few minutes ago on a new machine, and the problem is still there:

```
% zonemaster-cli toto.fr
Seconds Level     Message
======= ========= =======
  19.97 WARNING   All nameservers have IPv4 addresses in the same AS (16276).
  19.97 WARNING   All nameservers are in the same AS (16276).
  20.08 NOTICE    192.5.4.2 returned no DS records for toto.fr.
  21.24 NOTICE    Delegation from parent to child is not properly signed (no_ds).
  26.65 WARNING   When asked for SOA records on "Www.tOTo.fr" and "WwW.tOto.fR", nameserver dns.ovh.net/213.186.33.102 returns different answers.
  27.18 WARNING   When asked for SOA records on "Www.tOTo.fr" and "WwW.tOto.fR", nameserver ns.ovh.net/213.251.128.136 returns different answers.
  27.18 ERROR     When asked for SOA records on "www.toto.fr" with different cases, all servers do not reply consistently.
```

**Comment by sandoche2k** [comment]
@bortzmeyer Both the parent issue (CPAN) and the issue #422 has been fixed in the latest update (engine version 1.0.10)
